eng/service.proj

```diff
@@ -9,14 +9,15 @@
 
   <ItemGroup>
     <ExcludeMgmtLib Include="..\sdk\*\*.Management.*\**\*.csproj;..\sdk\*mgmt*\**\*.csproj;" />
+    <ExcludeTrack1Projects Include="..\sdk\*\Microsoft.*\**\*.csproj" Condition="'$(ExcludeTrack1)'=='true'" />
     <TestProjects Include="..\sdk\$(ServiceDirectory)\**\tests\**\*.csproj" />
     <SamplesProjects Include="..\sdk\$(ServiceDirectory)\**\samples\**\*.csproj" />
     <SampleApplications Include="..\samples\**\*.csproj" />
     <SrcProjects Include="..\sdk\$(ServiceDirectory)\**\*.csproj" Exclude="@(TestProjects);@(SamplesProjects)"/>
-    <ProjectReference Include="@(TestProjects)" Exclude="@(ExcludeMgmtLib)" Condition="'$(IncludeTests)' == 'true'" />
-    <ProjectReference Include="@(SamplesProjects)" Exclude="@(ExcludeMgmtLib)" Condition="'$(IncludeSamples)' == 'true'" />
-    <ProjectReference Include="@(SampleApplications)" Exclude="@(ExcludeMgmtLib)" Condition="'$(IncludeSamples)' == 'true'"/>
-    <ProjectReference Include="@(SrcProjects)" Exclude="@(ExcludeMgmtLib)" Condition="'$(IncludeSrc)' == 'true'" />
+    <ProjectReference Include="@(TestProjects)" Exclude="@(ExcludeMgmtLib);@(ExcludeTrack1Projects)" Condition="'$(IncludeTests)' == 'true'" />
+    <ProjectReference Include="@(SamplesProjects)" Exclude="@(ExcludeMgmtLib);@(ExcludeTrack1Projects)" Condition="'$(IncludeSamples)' == 'true'" />
+    <ProjectReference Include="@(SampleApplications)" Exclude="@(ExcludeMgmtLib);@(ExcludeTrack1Projects)" Condition="'$(IncludeSamples)' == 'true'"/>
+    <ProjectReference Include="@(SrcProjects)" Exclude="@(ExcludeMgmtLib);@(ExcludeTrack1Projects)" Condition="'$(IncludeSrc)' == 'true'" />
   </ItemGroup>
 
   <Import Project="..\sdk\$(ServiceDirectory)\*.projects" />
```

Comment:
If there is a need just for search I'd suggest using the service.projects model to exclude projects. See https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/batch/service.projects as an example.

Reply:
I imagine this will be more prevalent as we migrate services. Alex's thought is that those responsible for track 1 are responsible for their own infrastructure/changes (even if that's tying into what we have). It's different for Key Vault because we own track 1 code in that case.

Reply:
Also, I don't see how that file is used. I found nothing in a GitHub or Code (find in files) search. Could you or @chidozieononiwu elaborate? Is it getting pulled in from another repo in the pipeline?

Reply:
Right now it is only made use of in the batch service https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/batch/service.projects

Reply:
Yes, but how does it get used? Right now I just see a file with nothing referencing it. Is there a wildcard pattern match to sdk/**/*.projects or something?

Reply:
(Discussed offline and I understand how it's used now. I'll make the change as suggested.)